content/plugins/ipam/dhcp.md

```diff
@@ -25,7 +25,12 @@ $ ./dhcp daemon
 
 If given `-pidfile <path>` arguments after 'daemon', the dhcp plugin will write
 its PID to the given file.
```

Comment:
```suggestion
its PID to the given file.  
```
Adding 2 spaces at the end of the line will tell the markdown parser to add a newline.